### PR TITLE
Add in-game options menu options for lightshafts and bloom

### PR DESCRIPTION
**Pull request by jg18**
***This is experimental work.***

Intended to address Issue #3375.

### CODE REVIEW

File: code/graphics/post_processing.h [hunk 1]
```diff
@@ -56,12 +56,19 @@ class PostProcessingManager {
 	const lightshaft_parameters& getLightshaftParams() const;
 	lightshaft_parameters& getLightshaftParams();
 
+	bool bloomShadersOk() const { return m_bloomShadersOk; }
+	void setBloomShadersOk(bool ok) { m_bloomShadersOk = ok; }
```
Comment (asarium) [inline code/graphics/post_processing.h:60]: Function bodies should be in the source file.

**Review by asarium — CHANGES_REQUESTED** [review]

**Comment by jg18** [comment]
Still pending verification from Mjn. I sent him a link to a test build over HLP Forums PM.

Since the latest commit doesn't affect behavior, I won't update the test build.

File: code/graphics/post_processing.cpp [hunk 2]
```diff
@@ -34,6 +35,31 @@ PostEffectUniformType mapUniformNameToType(const SCP_string& uniform_name)
...
+// used by In-Game Options menu
+bool Post_processing_enable_lightshafts = true;
+
+auto LightshaftsOption =
+	options::OptionBuilder<bool>("Graphics.Lightshafts", "Lightshafts", "Enable lightshafts (requires post-processing)")
+		.category("Graphics")
+		.default_val(true)
+		.level(options::ExpertLevel::Advanced)
+		.bind_to(&Post_processing_enable_lightshafts)
+		.importance(60)
```
Comment (jg18) [inline code/graphics/post_processing.cpp:48]: The importance values are total guesses. I presume they define the order of options in the menu?

File: code/graphics/post_processing.cpp [hunk 3]
```diff
@@ -34,6 +35,31 @@ PostEffectUniformType mapUniformNameToType(const SCP_string& uniform_name)
...
+		.finish();
+
+int Post_processing_bloom_intensity = 25; // using default value of Cmdline_bloom_intensity
+
+auto BloomIntensityOption = options::OptionBuilder<int>("Graphics.BloomIntensity",
+	"Bloom intensity",
+	"Set bloom intensity (requires post-processing)")
+	.category("Graphics")
+	.range(0, 200)
```
Comment (jg18) [inline code/graphics/post_processing.cpp:57]: The [FS Wiki says](https://wiki.hard-light.net/index.php/Command-Line_Reference#-bloom_intensity) the range for `-bloom_intensity` is 0-200, but the call site for `gr_bloom_intensity()` in render_options.cpp makes me wonder if it's really 0-100.

**Review by jg18 — COMMENTED** [review]
A couple things to call out for discussion.

File: code/graphics/post_processing.cpp (hunk 2 above)
Comment (asarium) [reply code/graphics/post_processing.cpp:48]: Yes, that is what they are used for in the existing UI. The actual values do not matter and are only compared between items in the same category.

**Review by asarium — APPROVED** [review]

**Pull request merged**